juriscraper/opinions/united_states/state/neb.py

```diff
@@ -11,33 +11,29 @@ def __init__(self, *args, **kwargs):
         self.status = "Published"
```

Comment:
I was looking at nebctapp, and some of the opinions without a citation have the notice:
> NOTICE: THIS OPINION IS NOT DESIGNATED FOR PERMANENT PUBLICATION AND MAY NOT BE CITED EXCEPT AS PROVIDED BY NEB. CT. R. APP. P. § 2-102(E), 

so their status should be "Unpublished"?

![image](https://github.com/freelawproject/juriscraper/assets/14970769/76116f68-36d9-43fc-8533-5f28ad8aaefe)

Reply:
this is great thanks